main.go

```diff
@@ -92,6 +104,18 @@ func main() {
 	signal.Notify(signals, os.Interrupt)
 
 	L.Info(fmt.Sprintf("pg2kafka[commit:%s] is now listening to notifications", Version))
+
+	// http server
+	routes := http.LoadDefaultRoutes()
+	var httpDone = make(chan struct{})
+	go func() {
+		err = serveHTTP(httpPort, routes, httpDone)
+		if err != nil {
+			L.Error("http server errored out")
+		}
+		close(httpDone)
+	}()
+
 	waitForNotification(listener, producer, eq, signals)
```

Comment:
This point seems save to consider the pg2kafka instance as ready to handle DB events.